### PR TITLE
Next fixes

### PR DESCRIPTION
**Pull request by jeremie-lucca**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1701

File: packages/ng/option/src/lib/item/tree-option-item.component.scss [hunk 1]
```diff
@@ -1,198 +1,194 @@
...
+    }
+
+    .optionItem-icons {
+      opacity: 1;
+    }
+  }
+}
+
+.optionItem-value {
+  padding: var(--components-options-item-padding-vertical) var(--components-options-item-padding-horizontal) var(--components-options-item-padding-vertical) .5rem;
```
Comment (BertrandPodevin) [inline packages/ng/option/src/lib/item/tree-option-item.component.scss:17]: `padding: var(--components-options-item-padding-vertical) var(--components-options-item-padding-horizontal);` ?

Comment (jeremie-lucca) [reply packages/ng/option/src/lib/item/tree-option-item.component.scss]: Indeed. I'm not sure why it has a specific style like this. Maybe Option item horizontal padding was larger in the past 🤔

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1701

**Review by BertrandPodevin — APPROVED** [review]

**Pull request merged**